applications/openshift/classification-banner/rule.yml

```diff
@@ -0,0 +1,30 @@
+prodtype: ocp4
+
+title: 'Enable Classification Banner on OpenShift Console'
+
+description: |-
+    A classification banner can be configured by creating a Console Notification CRD on OpenShift
+
+rationale: |-
+    Displays to users organization-defined system use notification message or banner before granting access to the system that provides privacy and security notices consistent with applicable federal laws
+
+identifiers: {}
+
+severity: medium
+
+references:
+    nist: AC-8
+
+warnings:
+- general: |-
+    {{{ openshift_cluster_setting("/apis/console.openshift.io/v1/consolenotifications/classification-banner") | indent(4) }}}
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
+    entity_check: "at least one"
+    filepath: /apis/console.openshift.io/v1/consolenotifications/classification-banner
+    yamlpath: ".spec.text"
+    values:
+    - value: "Unclassified"
```

Comment:
this seems to only verify that the banner has the text "Unclassified" is this expected? Wouldn't we want to verify that folks have put at least something? you could use a regex here.

Reply:
@JAORMX This is expected! The text itself is custom depending on what the user set as the value. I'm open to a better way if you think I am missing something. The value could be `text: Anything`. I chose `text: Unclassified` as this rule is expecting a CRD is created, via the `e2e-remediation.sh`, that creates the CRD with `text: Unclassified`

https://docs.openshift.com/container-platform/4.7/web_console/customizing-the-web-console.html#creating-custom-notification-banners_customizing-web-console

Reply:
So, this will try to match "Unclassified" for the text and will fail on any other value. I think you instead want to use a regex to indicate that you want to match anything. So let's put:

```
    - value: ".*"
      operation: "pattern match"
      type: "string"
```

Here `.*` will be the thing that matches any set of characters.